src/app/app-components/duration-input-component/duration-input-component.component.html

```diff
@@ -1,2 +1,2 @@
-<input type="number" placeholder="minutes" [(ngModel)]="minutes" min="1">
-<span *ngIf="minutes">{{minutes | duration}}</span>
\ No newline at end of file
+<input placeholder="minutes" [(ngModel)]="minutes" [disabled]="disabled">
+<span *ngIf="minutes && !isValidLength">{{minutes | duration}}</span>
```

Comment:
Shouldn't we show it when the length is valid instead?

Reply:
good catch